tensorhive/models/Restriction.py

```diff
@@ -217,11 +217,11 @@ def as_dict(self, include_groups=False, include_users=False, include_resources=F
             'schedules': [schedule.as_dict for schedule in self.schedules]
         }
         if include_groups:
-            restriction['groups'] = [group.as_dict for group in self.groups]
+            restriction['groups'] = [group.as_dict(include_users=False) for group in self.groups]
         if include_users:
             # using as_dict_shallow as we do not want to include user's groups here, as they are insignificant
             # considering the scope of the restriction
```

Comment:
This comment is currently seems unnecessary

Reply:
fixed in ad81057cbbe9e68b203cf6756f2392cfdd816344